files/synchronize.py

```diff
@@ -164,6 +164,11 @@
     default: no
     required: false
     version_added: "2.0"
+  ssh_passwd:
+    description:
+      - Tells rsync to use sshpass with password
+    required: false
+    version_added: "2.0"
```

Comment:
I still believe this should not be a module argument, at least not one that accepts an actual password.  Since we know the password already if we are performing all other actions via password, we should be able to inject this via the `synchronize` action plugin in ansible itself.